### PR TITLE
Add dashboard example

### PR DESCRIPTION
**Pull request by brancz**
@metalmatze

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,70 @@ $ jsonnet -J vendor example.jsonnet
...
+local servicePort = k.core.v1.service.mixin.spec.portsType;
+
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local row = grafana.row;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+local graphPanel = grafana.graphPanel;
+
+local grafana = ((import 'grafana/grafana.libsonnet') + {
```
Comment (metalmatze) [inline README.md]:
```suggestion
local grafana = 
  ((import 'grafana/grafana.libsonnet') + {
```
I've previously talked to @squat about this: Can we make a new line so we don't have this weird indentations?
/cc @s-urbaniak

File: examples/dashboard-definition.jsonnet [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+local servicePort = k.core.v1.service.mixin.spec.portsType;
+
+local grafana = import 'grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local row = grafana.row;
+local prometheus = grafana.prometheus;
+local template = grafana.template;
+local graphPanel = grafana.graphPanel;
+
+local grafana = ((import 'grafana/grafana.libsonnet') + {
```
Comment (metalmatze) [inline examples/dashboard-definition.jsonnet]:
```suggestion
local grafana =
  ((import 'grafana/grafana.libsonnet') + {
```

File: README.md (hunk 1 above)
Comment (s-urbaniak) [reply README.md]: :+1: I believe I follow the proposed idiom already here: https://github.com/openshift/cluster-monitoring-operator/commit/534c4fe68a64708253e6cdc518a049ab3886067d#diff-254ce52fe46e44a39a4644a7bf612a2

Comment (brancz) [reply README.md]: yes makes sense, please take a look

**Pull request merged**